src/librustc_mir/transform/inline.rs

```diff
@@ -101,6 +103,7 @@ impl Inliner<'tcx> {
                 }
 
                 let callee_body = if let Some(callee_def_id) = callsite.callee.as_local() {
+                    let _ = self.tcx.mir_inliner_cycle(callee_def_id);
```

Comment:
The inliner is not enabled, so this will never run.

Reply:
considering that clap and wf-projection-stress regressed and loads of crates took a <1% hit, I think at least the perf bot ran it.

Reply:
The incremental benches are most affected. Maybe the new query caused a slowdown? `rg "query .*\(" src/librustc_middle/query/mod.rs | wc -l` on an older checkout returns `211`. Maybe the amount of queries got pushed over `256`? (or some lower number if there are reserved discriminants)

Reply:
I get `214`. I'll move the query invocation somewhere where it'll get invoked for every function